README.md

```diff
@@ -16,27 +16,40 @@ To install the operator courier project to use from the command line, just insta
 $ pip3 install operator-courier
 ```
 
-Once the project is installed, you can verify that a set of files is valid and can be bundled and pushed. First, create a flat directory containing all of the CSV, CRD and PACKAGE files that are included in your bundle.  Then just use `operator-courier verify` to test it.
+Once the project is installed, you can run `verify` command on a set of operator manifests with different versions in the source directory. The source directory includes a package file in the source directory root, and version folders containing CRD and CSV files required for that manifest version (See [Manifest format](https://github.com/operator-framework/operator-registry#manifest-format) for more details)
 
 ```bash
-$ operator-courier verify $BUNDLE_DIR
+$ operator-courier verify $MANIFESTS_DIR
 ```
 
-To generate an operator bundle and push it to a quay.io app registry just use `operator-courier push`. Just pass the directory, namespace, repository, release version and quay.io authorization token needed to push.
+To push the operator manifests to a quay.io app registry just use `operator-courier push`. Just pass the directory, namespace, repository, release version and quay.io authorization token needed to push.
 
 ```bash
-$ operator-courier push $BUNDLE_DIR $EXAMPLE_NAMESPACE $EXAMPLE_REPOSITORY $EXAMPLE_RELEASE "$AUTH_TOKEN"
+$ operator-courier push $MANIFESTS_DIR $EXAMPLE_NAMESPACE $EXAMPLE_REPOSITORY $EXAMPLE_RELEASE "$AUTH_TOKEN"
 ```
 
 Once that is created, you should be able to view your bundle on quay.io's Application page for your particular namespace, repo, and release version (https://quay.io/application/$EXAMPLE_NAMESPACE/$EXAMPLE_REPOSITORY?tab=$EXAMPLE_RELEASE)
```

Comment:
`... **bundle** on quay.io's Application ...`

Should we use the term `bundle` here?

Reply:
Thanks @awgreene. Great catch!